vpn-proxy/app/views.py

```diff
@@ -17,19 +21,25 @@ def __init__(self, data, **kwargs):
 def tunnels(request):
     if request.method == 'POST':
         params = {}
-        if 'server' in request.POST:
-            params['server'] = request.POST['server']
+        cidrs = request.POST.getlist('cidrs')
```

Comment:
we should be using this list of cidrs to:
1. store them in the tunnel model
2. put them in the tunnel's routing table instead of a default gw
3. check that forwarded ip's match the cidr's before creating them